### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by zephyr-zeph** [Lsoissus/davebambimidi#2] [Ideas]
You can discuss what midis i should add in, [Must be dave and bambi related song!]
Or the things you experience.

**Comment by zephyr-zeph** [comment]
test

**Comment by xmati-swag** [comment]
https://user-images.githubusercontent.com/93664892/158756738-9c020764-c637-4582-adda-e03db5c46a8e.mp4

**Comment by GeneralQwar** [comment]
can you make house midi

**Reply by zephyr-zeph** [reply]
ok
